ansible_base/resource_registry/views.py

```diff
@@ -95,8 +112,34 @@ def manifest(self, request, name, *args, **kwargs):
 
 
 class ServiceMetadataView(AnsibleBaseDjangoAppApiView):
-    permission_classes = [permissions.IsAuthenticated]
+    permission_classes = [
+        HasResourceRegistryPermissions,
+    ]
+    action = "retrieve"
 
     def get(self, request, **kwargs):
         registry = get_registry()
         return Response({"service_id": service_id(), "service_type": registry.api_config.service_type})
+
+
+class ValidateLocalUserView(AnsibleBaseDjangoAppApiView):
```

Comment:
I would suggest adding a long docstring here, because that should get rendered in the Browsable API, and make its way into swagger and stuff. Because I have a hard time figuring out what the function of this endpoint is.